tests/x11/x11_login.pm

```diff
@@ -4,11 +4,38 @@ use testapi;
 sub run() {
     my $self = shift;
 
-    # log in
-    type_string $username. "\n";
-    sleep 1;
-    type_string $password. "\n";
+    save_screenshot;
+
+    # make sure back to login screen
+    send_key "ctrl-alt-backspace";
```

Comment:
do this force logout just because, we already logged in to desktop environment at first_boot, thus once console tests finished and move to x11 test, we're just switch back to desktop environment, and I don't think do a reboot again is a good idea due to we will test it later. and per main.pm, this test only execute if NOAUTOLOGIN or XDMUSED sets, however I've go through all cases, I guess this test actually never used, minimalx never do x11 test; similar reason for rescue test(xfce), it is logged in already, typing username and password(original content here) means nothing...

hmm...but I can change the way to use right click menu ->logout, is it better?